src/app/public/modules/radio/radio-group.component.ts

```diff
@@ -37,20 +37,38 @@ import {
 
 let nextUniqueId = 0;
 
+/**
+ * The `sky-radio-group` component organizes radio buttons into a group. It is required for radio
+ * buttons on Angular reactive forms, and we recommend using it with all radio buttons.
+ * On Angular forms, the component manages the selected values and keeps the forms up-to-date.
+ * When users select a radio button, its value is driven through an `ngModel` attribute that you specify on the `sky-radio-group` element.
+ */
 @Component({
   selector: 'sky-radio-group',
   templateUrl: './radio-group.component.html'
 })
 export class SkyRadioGroupComponent implements AfterContentInit, AfterViewInit, OnDestroy {
 
+/**
+ * Sets the radio button group's `aria-labelledby` attribute to support
+ * [accessibility](https://developer.blackbaud.com/skyux/components/radio#accessibility). The value
```

Comment:
This link is busted. I think we can just remove the link, but I made a note elsewhere about how to incorporate the *Accessibility* section content into property descriptions.

Reply:
✔️

Reply:
Link looks good, but I'd still incorporate content from the *Accessibility* section into this description unless you decide to pursue setting up the entire section somewhere else.

Reply:
Added the Accessibility section but removed the link to the section.